src/extension.ts

```diff
@@ -47,21 +47,21 @@ const getOpenFileURI = () => {
     vscode.window.showErrorMessage('No editor active');
     return undefined;
   }
-}
+};
 
 /**
  * Opens a terminal and sends it a command
  */
 const sendAndFocus = (command: string) => () => {
```

Comment:
Why the unit currying here?

Reply:
I was also wondering about this. It looks like `registerCommand` expects a callback: https://code.visualstudio.com/api/references/vscode-api#commands.registerCommand, and we are configuring the callback with some extra arguments.

Reply:
Hmmm, I'd suggest wrapping in `() =>` at the call site in that case. It feels weird to need to pass this extra `()` all over the place, no?